net-firewall/firewalld/firewalld-2.1.1-r2.ebuild

```diff
@@ -167,6 +167,7 @@ src_prepare() {
 	sed -i \
```

Comment:
I think I'd prefer a patch to avoid us leaving junk in there.

Reply:
I've replaced the sed command with a separate patch file.